wazuh/wazuh-dashboard-plugins
# Shown Wazuh API version is wrong or empty after upgrade

### Description

During https://github.com/wazuh/wazuh/issues/27183, it was found that Wazuh API version shown is not the current one after upgrading from v4.9.2 to v4.10.0-rc1

![image](https://github.com/user-attachments/assets/22f31553-ed97-4a19-b81d-6962a5fa00bc)

After cleaning the cache, the error is worse

![image](https://github.com/user-attachments/assets/217a5f75-54cd-4868-a69a-1b0ec24dea86)

**Expected behavior**

Wazuh API is v4.10.0

![image](https://github.com/user-attachments/assets/54d2ba0e-2c8d-4dfe-a3d3-10c065a13a18)

### Issues Resolved

Closes https://github.com/wazuh/wazuh-dashboard/issues/440

### Evidence

![image](https://github.com/user-attachments/assets/0822bafa-df38-4dd7-a0db-e0f219ce00ad)

![image](https://github.com/user-attachments/assets/613dc286-1fe5-4711-97b1-b0dafc1c0c52)

![image](https://github.com/user-attachments/assets/6164fc74-0925-438f-b596-0b623e5a5cda)

![image](https://github.com/user-attachments/assets/1476173a-36e1-429c-9cca-aaf3592e4e02)


### Test

1. Navigate to `Dashboard management` » `Server APIs`
2. Open DevTools
3. Go to `Application` » `Session storage`
4. Delete key `checkUpdates`
  ![image](https://github.com/user-attachments/assets/a2b98f41-90f0-4bb5-be98-6f269d362472)
5. Reload the page
6. Please verify the status of the `Version` column under two scenarios: when everything is correct and when an issue arises.

> [!NOTE]
> Remove `checkUpdates` for each reloaded page.

#### Unit tests

![image](https://github.com/user-attachments/assets/5a72b57c-a368-4fe0-8aa2-8036327d40ac)

### Check List
